### ISSUE TITLE
Can't find -lXxf86vm when I try install GLFW 

### ISSUE THREAD

**Issue by LuisFlahan4051** [go-gl/glfw#286]
github.com/go-gl/glfw/v3.3/glfw
/usr/bin/ld: Can't find -lXxf86vm

**Comment by pwaller** [comment]
What platform are you developing on? Have you followed the installation instructions in the readme? https://github.com/go-gl/glfw#installation

**Comment by LuisFlahan4051** [comment]
Of course, I followed the instructions of readme.
I had already installed it before in Ubuntu but right now I'm on Fedora.

When I enter the browser at the address github.com/go-gl/glfw/v3.3/glfw it gives 404 error.
I saw that it is in the master branch https://github.com/go-gl/glfw/tree/master/v3.3/glfw, I tried to install it from there but it doesn't work, I don't know what to do.

Thank you so much for the support.
